### ISSUE TITLE
AudioCache prefix is changed since `audioplayers` 0.16.0

### ISSUE THREAD

**Issue by mpugach** [flame-engine/flame#657]
## Description

Since `audioplayers` version 0.16.0 the mandatory non-configurable prefix is changed
https://github.com/luanpotter/audioplayers/blob/0.15.1/lib/audio_cache.dart#L138
https://github.com/luanpotter/audioplayers/blob/0.16.0/lib/audio_cache.dart#L147

This causes `Unable to load asset` exception for assets in `./assets/audio`

## Development environment

Flame version:

0.29.0
0.29.1-beta

0.28.0 is not affected since it depends on `audioplayers: ^0.15.1`

Flutter doctor:

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 1.22.6, on macOS 11.2.1 20D74 darwin-x64, locale en-UA)

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 12.4)
[!] Android Studio (version 4.1)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[✓] VS Code (version 1.53.1)
[✓] Connected device (1 available)
```

## Runtime

This issue is related to running on which platform? (Select all that apply)

 - [x] Android :android:
 - [x] iOS :apple:
 - [ ] Web :browser: **(Did not tried)**

## Minimal reproducible code (Required for bugs)

https://github.com/japalekhin/langaw/tree/22f064e91f42f88198ab3473718662a778b828b5 just change it to 0.29.0 in the pubspec.yaml

**Comment by luanpotter** [comment]
Excellent catch, @mpugach, I indeed forgot about this small breaking change I did on AP.

Fix: https://github.com/flame-engine/flame/pull/665

**Comment by erickzanardo** [comment]
Fixed on #665
